fashion-messaging-app-ui/src/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,40 @@
+import { Link, useNavigate } from 'react-router-dom';
+import './LoginForm.css';
+
+const LoginForm = () => {
+  const navigate = useNavigate();
+
+  const handleLogin = (e) => {
+    e.preventDefault();
+
+    navigate('/');
+  };
+
+  return (
+    <div className='login-form-container'>
+      <form className="login-form" onSubmit={handleLogin}>
+        <h2>Login</h2>
+        <div className="form-group">
+          <label htmlFor="username">Username:</label>
+          <input
+            type="text"
+            id="username"
+          />
+        </div>
+        <div className="form-group">
+          <label htmlFor="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+          />
+        </div>
```

Comment:
There may have been a way to share more code between Login and Signup since the UI's are quite similar and the Username and Password parts are the same

Reply:
> There may have been a way to share more code between Login and Signup since the UI's are quite similar and the Username and Password parts are the same

Hmm could you please elaborate on this?